content/docs/guides/LSAAI/Configure_LSAAI.md

```diff
@@ -0,0 +1,52 @@
+# Configure KeyCloak with Azure and LSAAI
+
+* * *
+
+## Keycloak
+
+For Keycloak, I utilized a Linux app service plan within Azure. One of the primary advantages is that HTTPS is readily configured out of the box, eliminating the need for additional work with certbot. Moreover, when using an app service plan, there's no need to be concerned about Linux updates.
```

Comment:
for documentation passive voice is preferable, e.g.: A Linux app service plan within Azure was utilised for Keycloak set up